### PR TITLE
Menu style

### PR DESCRIPTION
**Pull request by bsfsnehal**
### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/Testing/menuhovertest.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+
+// import { createURL } from "@wordpress/e2e-test-utils";
+// import { setCustomize } from "../../utils/set-customize";
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/Testing/menuhovertest.js:3]: Remove commented code

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/Testing/menuhovertest.js:3]: yes

File: tests/e2e/specs/customizer/Testing/menuhovertest.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+// 			waitUntil: 'networkidle0',
+// 		} );
+//         await page.waitForSelector( '#ast-hf-menu-1' );
+//         //ast-select-input
+        
+//     });
+//     })
+import { setCustomize } from '../../../utils/set-customize';
+describe( 'breadcrumb Typography settings in the customizer', () => {
+    it( 'breadcrumb typography should apply corectly', async () =>
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/Testing/menuhovertest.js:25]: Why these breadcrumb related changes are there in menu file?

**Comment by sushmak02** [comment]
Remove unused files and commented code. This PR should contain only a single file for your tescases.

**Pull request closed without merging**